### PR TITLE
allow specifying more themes, comma separated  

### PR DESCRIPTION
**Pull request by cliviu**
https://github.com/vaadin/maven-plugin/issues/2

### CODE REVIEW

File: src/main/java/com/vaadin/integration/maven/AbstractThemeMojo.java [hunk 1]
```diff
@@ -138,13 +138,19 @@ protected File configureThemeClasspath(JavaCommand cmd, String theme) throws Moj
...
      * @return the theme names
      */
     protected String[] getThemes() {
         // theme has higher priority if set by expression
         if (theme != null) {
-            return new String[] { "VAADIN/themes/" + theme };
+            String[] themes = theme.split(",");
+            String[] themesPaths = new String[themes.length];
+            for(int i = 0; i < themesPaths.length ; i++) {
+                themesPaths[i] = "VAADIN/themes/" + themes[i];
```
Comment (denis-anisimov) [inline src/main/java/com/vaadin/integration/maven/AbstractThemeMojo.java]: It's worth to remove extra spaces here using `trim()`.

File: src/test/java/com/vaadin/integration/maven/AbstractThemeMojoTest.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        extends AbstractMojoTestCase {
+
+    public void testGetThemes() throws Exception {
+        CompileThemeMojo compileThemeMojo = new CompileThemeMojo();
+
+        setVariableValueToObject( compileThemeMojo, "theme", "theme1");
+        assertEquals(1,compileThemeMojo.getThemes().length);
+        assertEquals("VAADIN/themes/theme1",compileThemeMojo.getThemes()[0]);
+
+        setVariableValueToObject( compileThemeMojo, "theme", "theme1,theme2");
```
Comment (denis-anisimov) [inline src/test/java/com/vaadin/integration/maven/AbstractThemeMojoTest.java]: And the test can be extended to have spaces around theme name.

**Comment by cliviu** [comment]
cool, thanks

**Comment by denis-anisimov** [comment]
You are welcome.
Thank you for your contribution.

**Pull request merged**